backend/endpoints/responses/rom.py

```diff
@@ -121,6 +121,7 @@ class RomSchema(BaseModel):
     # Metadata fields
     first_release_date: int | None
     youtube_video_id: str | None
+    average_rating: float
```

Comment:
I would ideally mark this field as nullable. In case a game doesn't match any metadata providers, it would be better for the frontend to show an empty rating rather than zero, which could be confusing for the user.